media/Jsontracer/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- 
```

Comment:
It may not be possible to insert this at the top of file, am I right?

Reply:
[here](https://github.com/catapult-project/catapult/blob/444aba89e1c30edf348c611a9df79e2376178ba8/tracing/tracing_examples/trace_viewer.html), put it under the html tag to mark the license in the browser.